components/apimgt/org.wso2.carbon.apimgt.rest.api.admin.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/admin/v1/impl/ThrottlingApiServiceImpl.java

```diff
@@ -523,6 +535,84 @@ public Response throttlingPoliciesSubscriptionPost(String contentType, Subscript
         return null;
     }
 
+    /**
+     * Returns list of throttling policy details filtered using query parameters
+     *
+     * @param query filtering parameters
+     * @return Retrieves Throttle Policies List
```

Comment:
Remove the word "Retrieves"

Reply:
Fixed with 30d0ecb9c29c0024fa182ba597c8eda4c128c451